lib/puppet/functions/nested_values.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+# This function will return list of Hash values, the return value will be Array
+# NOTE : This function is expecting only Hash and return value will be Array
```

Comment:
These two lines could be reworded to remove the repetition about the return value.

Reply:
addressed [here](url https://github.com/puppetlabs/puppetlabs-stdlib/pull/1363)